internal/staticupload/staticupload_test.go

```diff
@@ -108,12 +108,13 @@ func TestUpload(t *testing.T) {
 			}
 			_, err := client.Upload(context.Background(), tc.in)
 
+			var invalidationErr *InvalidationError
 			if tc.wantCacheInvalidationErr {
-				assert.ErrorAs(err, &InvalidationError{})
+				assert.ErrorAs(err, &invalidationErr)
 				return
 			}
 			if tc.wantErr {
-				assert.False(errors.As(err, &InvalidationError{}))
+				assert.False(errors.As(err, &invalidationErr))
 				assert.Error(err)
 				return
 			}
```

Comment:
```suggestion
			var invalidationErr *InvalidationError
			if tc.wantCacheInvalidationErr {
				assert.ErrorIs(err, &invalidationErr)
				return
			}
			if tc.wantErr {
				assert.ErrorIs(err, &invalidationErr))
				assert.Error(err)
				return
			}
```

Reply:
ok we have it in several places replicated.. i guess it doesn't matter

Reply:
I'm not fully understanding the suggestion?
The current code makes sure that if `tc.wantErr` is true, but `tc.wantCacheInvalidationErr` is false, then we check that the returned error **is not** of type `*InvalidationError`, your suggestion would change that to always checking that it is